LINUX/packaging/netmap.spec

```diff
@@ -0,0 +1,60 @@
+Name:          netmap
+Version:       %{version}
+Release:       %{buildnumber}%{?dist}
+Summary:       netmap for Network Monitor
+Group:         Development/Tools
+License:       Dual BSD/GPL
+BuildRequires: kernel-devel = 4.18.0-193.19.1.el8_2, kernel = @KERNEL_VERSION@
+Requires:      kernel = @KERNEL_VERSION@
+ExclusiveArch: x86_64
+
+%description
+
+%prep
+cd ~/rpmbuild/BUILD
+/bin/mkdir %{name}
+cd %{name}
+/bin/tar xzf ~/rpmbuild/SOURCES/%{name}-%{version}.%{buildnumber}.tar.gz
+if [ $? -ne 0 ]; then
+   exit $?
+fi
+
+%build
+cd %{name}/LINUX/
+./configure --no-drivers --kernel-dir=/usr/src/kernels/4.18.0-193.19.1.el8_2.x86_64/
```

Comment:
I had to hard code this path to get the build to work. The path does not match the output from `uname -r`. Compare:
4.18.0-193.19.1.el8_2.x86_64
4.18.0-193.el8.x86_64